README.md

```diff
@@ -100,6 +100,10 @@ By default, Upgrade Responder only groups data by `appVersion` field and creates
 If you add extra fields and want to display statical information for those fields, there are extra steps you need to follow to setup InfluxDB and Grafana. 
 See [Add kubernetesVersion extra field](#add-kubernetesVersion-extra-field) for an example of how to add an extra field. 
 
+If your application is written in Golang, you can import our provided [client package](./client) and use it to save time writing code. 
```

Comment:
Add a subtitle to give this section more visibility. Otherwise users might missed this.

Reply:
Thank you. added